dasharo/README.md

```diff
@@ -5,7 +5,7 @@
 For details about Dasharo keys naming please read
 [documentation](https://docs.dasharo.com/dev-proc/versioning/#signing-keys).
 
-```
+```shell
 $ gpg --expert --full-gen-key --allow-freeform-uid
```

Comment:
@macpijan I don't like this format, although it is a lot of work to fix. Can we force following format:

* Command
   ```shell
   command
   ```
* Output
   ```
   command output
   ```

The goal is to improve copypastability and usefulness of documentation.

Reply:
@pietrushnic You mean to just remove the `$` from code block?

Reply:
There is this rule for that: 
https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output

Reply:
But it cannot enforce splitting blocks into two, if both command and output are already in one block. This would need to be split manually.

Reply:
I enabled the check here: 9e784b9d7b05 and apllied a few autofixes, but the majority must be done manually if we want to.

Reply:
@macpijan are you saying we need a separate issue for splitting those blocks because this PR is only about applying pre-commit?

Reply:
@pietrushnic If we want to split those blocks, then yest we need another issue/MR. I am not really willing to do that right now, as it seems quite a lot of that.

Reply:
Created issue to be addressed in future: https://github.com/3mdeb/3mdeb-secpack/issues/110